docs/contracts.rst

```diff
@@ -194,9 +194,16 @@ return parameter list for functions.
         function setData(uint a) internal { data = a; }
         uint public data;
```

Comment:
As discussed, please remove the `public` from the variable here and change to:

   function setData() internal { _data = a; }
   function getData() { return _data; }
   uint _data;